stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -139,7 +139,7 @@ public extension Tensor {
     // TF CPU first, before TF CPU sends the tensor to TPU.
     let ret: TensorHandle<Scalar> = #tfop("Identity",
                                           tensor,
-                                          __shapes: [shape],
+                                          __shapes$shapearray: [shape],
```

Comment:
I'm sorry for the back and forth on this, but now that SymbolicValue::Array carries with it the element type, we can model shapearrays as just normal attributes.  This will allow us to drop the "$shapearray" modifier when the legacy code goes away.  WDYT?

Reply:
Sounds good. This makes it more user friendly when user wants to use \#tfop directly. Done!